charts/backend-service/values.yaml

```diff
@@ -8,16 +11,22 @@ rollingUpdate:
   maxSurge: 1
   maxUnavailable: 0
 
+# -- The image to use for this specific deployment and its cron jobs
 image:
-  repository: ""
+  # -- Repository, where the Unique service image is pulled from
+  # - for Unique internal deployments, these is the internal release repository
+  # - for client deployments, this will refer to the client's repository where the images have been mirrored too
+  # Note that it is bad practice and not advised to directly pull from Uniques release repository
+  # Read in the readme on why the helm chart comes bundled with the unique-ag/chart-testing-service image
```

Comment:
I wonder if it is smart to make our repository the default then.

Reply:
We have to. Our real images are protected and can't be pulled or tested with.